mscore/exportdialog.cpp

```diff
@@ -519,8 +519,8 @@ void ExportDialog::accept()
                         success = false;
                   }
             }
-      if (success)
-            QMessageBox::information(this, tr("Export"), tr("Export was successful."));
+      if (!success)
+            QMessageBox::critical(this, tr("Export"), tr("Export was not successful."));
```

Comment:
If the export is *not* successful, `mscore->saveAs` already presents a warning to the user. So no need to do it here another time. 

@Tantacrul told me that he preferred to keep the "Export successful" message.

Reply:
Yes. I used it a few times recently and found that the dialog can hang for a little while (If I'm exporting a long audio file for example). At that point, the 'successful' dialog, although a bit dated, was reassuring. Otherwise the user may be led to believe the process failed. In future, it would be ideal to create a progress bar-type solution.

Reply:
For longer exports like MP3 we do have that progres bar. Having to dismiss this success dialog is highly annoying

Reply:
For longer exports like MP3 and WAV we do have that progres bar (and ever had)
Having to dismiss this success dialog is highly annoying

Reply:
reverted and commented out...